check-mirrors.sh

```diff
@@ -1,16 +1,20 @@
 #!/bin/sh
 
 temp_path="$(mktemp)"
-
+summary_file=${GITHUB_STEP_SUMMARY:-/tmp/check-mirror-summary.md}
```

Comment:
Why this `/tmp/check-mirror-summary.md` result? If running as an Action the env var should always be populated, and if running locally I definitely don't want it putting random stuff in my `/tmp`. I would accept defaulting to `check-mirror-summary.md` and putting it in the `.gitignore`.

Reply:
Fixed